versioned/mongodb/src/main/java/com/dremio/nessie/versioned/store/mongodb/MongoDBStore.java

```diff
@@ -159,12 +179,8 @@ public void start() {
     mongoDatabase = mongoClient.getDatabase(config.getDatabaseName());
 
     // Initialise collections for each ValueType.
-    collections = Stream.of(ValueType.values()).collect(ImmutableMap.<ValueType, ValueType, MongoCollection<? extends HasId>>toImmutableMap(
-        v -> v,
-        v -> {
-          String collectionName = v.getTableName(config.getTablePrefix());
-          return ((MongoCollection<? extends HasId>) mongoDatabase.getCollection(collectionName, v.getObjectClass()));
-        }));
+    TYPE_COLLECTIONS.forEach((k, v) ->
+        collections.put(k, (MongoCollection<? extends HasId>)mongoDatabase.getCollection(v.apply(config), k.getObjectClass())));
```

Comment:
Code looks much better here.
However, they need to be now hardcoded in TYPE_COLLECTIONS, while previously it would iterate over them.
Is that wanted/better/safer?